frontend/src/components/editor.js

```diff
@@ -11,7 +11,7 @@ export default function Editor({ setDisable, comment, presets, imagery, gpxUrl }
   const iDContext = useSelector((state) => state.editor.context);
   const locale = useSelector((state) => state.preferences.locale);
   const [customImageryIsSet, setCustomImageryIsSet] = useState(false);
-  const windowInit = typeof window !== undefined;
+  const windowInit = typeof window !== 'undefined';
```

Comment:
Isn't the `window` object always available? Is it possible for the `window` object to be `undefined` under any circumstances?

Reply:
I think it is always available in a browser. I don't think it is in NodeJS.